openstack/clustering/v1/nodes/doc.go

```diff
@@ -70,5 +70,19 @@ Example to Get a Node
 	}
 
 	fmt.Printf("%+v\n", node)
+
+Example to Perform an Operation on a Node
+
+	serviceClient.Microversion = "1.4"
+	nodeID := "node123"
+	operationOpts := nodes.OperationOpts{
+		Operation: nodes.RebootOperation,
+		Params:    nodes.OperationParams{"type": "SOFT"},
+	}
+    actionID, err := nodes.Ops(serviceClient, nodeID, operationOpts).Extract()
+	if err != nil {
+		panic(err)
+	}
+
```

Comment:
@JackKuei I'm so sorry - I didn't see this before. Looks like there's a formatting issue here.

Other than that, this LGTM and I can merge once fixed.

Reply:
Good catch! Making a fix